Makefile

```diff
@@ -394,6 +394,10 @@ email-mocks: ## Creates mocks for misc interfaces.
 	$(GO) install github.com/vektra/mockery/v2/...@v2.10.4
 	$(GOBIN)/mockery --dir app/email --name ServiceInterface --output app/email/mocks --note 'Regenerate this file using `make email-mocks`.'
 
+platform-mocks: ## Creates mocks for platform interfaces.
+	$(GO) install github.com/vektra/mockery/v2/...@v2.14.0
+	$(GOBIN)/mockery --dir app/platform --name SuiteIFace --output app/platform/mocks --note 'Regenerate this file using `make platform-mocks`.'
```

Comment:
Why is it called `SuiteIFace` when it generates platform mocks?

Reply:
It's mock for suite interface, The future suite service will going to provide that interface.

Reply:
Ah thanks